package.json

```diff
@@ -9,7 +9,7 @@
     "type": "git"
   },
   "author": "Mapbox",
-  "version": "2.7.1",
+  "version": "2.8.0",
```

Comment:
I think this'd be a patch update, and should ship alongside #142

Reply:
Will leave version changes to `master`